src/main/java/uk/gov/hmcts/sptribs/ciccase/model/UserRolesForAccessProfiles.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.sptribs.ciccase.model;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import uk.gov.hmcts.ccd.sdk.api.HasRole;
+
+@AllArgsConstructor
+@Getter
+public enum UserRolesForAccessProfiles implements HasRole {
```

Comment:
I am not sure whether case flags need these many things. Remove unwanted things from here.